src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/PublicationController.cs

```diff
@@ -62,11 +62,14 @@ public async Task<ActionResult<List<PublicationSummaryViewModel>>> ListPublicati
         }
 
         [HttpGet("api/publication/{publicationId}/releases")]
-        public async Task<ActionResult<List<ReleaseViewModel>>> ListActiveReleases(
-            [Required] Guid publicationId)
+        public async Task<ActionResult<PaginatedListViewModel<ReleaseListItemViewModel>>> ListActiveReleases(
+            [Required] Guid publicationId,
+            [FromQuery] int? page = null,
```

Comment:
Might be that as we add more pagination to the codebase that we have a `record` called something like `PaginationParams` which holds `page` and `pageSize` within it.

Then it also makes it easier to pass these multiple parameters around through from Controller methods to Service methods to Repository methods.

Also then makes it easier to add new pagination parameters in the future in one place if necessary.

Doesn't have to be done as part of this PR (or at all in fact), but might be worth considering.

Reply:
Or `PaginationRequest` rather than `PaginationParams` - it's a bit more explicit that it's something to do with a request rather than a response.

Reply:
Ok - I will leave it as it is for now.